Source/ASDisplayNode.mm

```diff
@@ -729,6 +729,8 @@ - (BOOL)_isNodeLoaded
 
 - (UIView *)view
```

Comment:
I think we have to add locking to this method ... any reason why we didn't have in the past?